src/components/gltf-model.js

```diff
@@ -0,0 +1,37 @@
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+/**
+ * glTF model loader.
+ */
+module.exports.Component = registerComponent('gltf-model', {
+  schema: {type: 'model'},
```

Comment:
I slightly prefer multiprop, but can't actually think of any others we'd need. Maybe crossorigin. I'm expecting animation to be a separate component, but if not that would definitely need some properties. I've used `src: {type: 'asset'}` in most of my loaders so far i think.